### PR TITLE
[IA-4067] Update GKM to 2.5.0 and fix job resource requests

### PR DESCRIPTION
**Pull request by afgane**
This updates GKM that uses the latest version of Galaxy (v23.0).

### CODE REVIEW

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala [hunk 1]
```diff
@@ -1507,8 +1507,8 @@ class GKEInterpreter[F[_]](
     // Machine type info
     val maxLimitMemory = machineType.memorySizeInGb
     val maxLimitCpu = machineType.numOfCpus
-    val maxRequestMemory = maxLimitMemory - 5
-    val maxRequestCpu = maxLimitCpu - 3
+    val maxRequestMemory = maxLimitMemory - 15
```
Comment (rtitle) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:1510]: Just curious about this - why increase the difference between requests & limits memory/cpu? (15G just seems like a lot.)

**Review by rtitle — APPROVED** [review]
I didn't test this but the change LGTM ✅ -- happy to merge if you all are happy with it.

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala (hunk 1 above)
Comment (afgane) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala:1510]: We added 2 new services to the chart (rabbitmq & tusd) and these were the values I saw on a cluster that had just started Galaxy.

**Comment by afgane** [comment]
I'd be glad to see this merged so we test on dev. (If the past month is any indication, there will be more iterations here but this is hopefully one big step closer to getting Galaxy running again.)

**Comment by lucymcnatt** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Review by jdcanas — APPROVED** [review]

**Comment by rtitle** [comment]
Not sure why `fiab-start` is failing -- jenkins retest

**Pull request merged**